trieste/acquisition/rule.py

```diff
@@ -94,27 +88,39 @@ def __init__(
         self,
         builder: AcquisitionFunctionBuilder | None = None,
         optimizer: AcquisitionOptimizer[SP_contra] | None = None,
+        num_query_points: int = 1,
     ):
         """
         :param builder: The acquisition function builder to use. Defaults to
-            :class:`~trieste.acquisition.ExpectedImprovement` with tag :data:`OBJECTIVE`.
+                :class:`~trieste.acquisition.ExpectedImprovement` with tag :data:`OBJECTIVE`.
         :param optimizer: The optimizer with which to optimize the acquisition function built by
-            ``builder``. This should *maximize* the acquisition function, and must be compatible
-            with the global search space. Defaults to
-            :func:`~trieste.acquisition.optimizer.optimize`.
+                ``builder``. This should *maximize* the acquisition function, and must be compatible
+                with the global search space. Defaults to
+                :func:`~trieste.acquisition.optimizer.optimize`.
+        :param num_query_points: The number of points to acquire.
         """
+
+        if not num_query_points > 0:
+            raise ValueError(
+                f"Number of query points must be greater than 0, got {num_query_points}"
+            )
+
         if builder is None:
-            builder = ExpectedImprovement().using(OBJECTIVE)
+            if num_query_points == 1:
+                builder = ExpectedImprovement().using(OBJECTIVE)
+            else:
+                raise ValueError("Need to specify a batch acquisition function")
 
         if optimizer is None:
             optimizer = optimize
 
         self._builder = builder
         self._optimizer = optimizer
+        self._num_query_points = num_query_points
 
     def __repr__(self) -> str:
         """"""
-        return f"EfficientGlobalOptimization({self._builder!r})"
+        return f"EfficientGlobalOptimization({self._num_query_points!r}, {self._builder!r})"
```

Comment:
I realise optimizer's already missing from this string, but the order needs to match the `__init__` args (unless using named parameters)